orbit/pkg/table/menubar/menubar.go

```diff
@@ -0,0 +1,64 @@
+//go:build darwin
+// +build darwin
+
+package menubar
+
+import (
+	"context"
+	"fmt"
+	"github.com/osquery/osquery-go/plugin/table"
+	"os"
+	"os/exec"
+	"strings"
+	"syscall"
+	"time"
+)
+
+// Columns is the schema of the table.
+func Columns() []table.ColumnDefinition {
+	return []table.ColumnDefinition{
+		table.IntegerColumn("location_services_enabled_presented"),
+	}
+}
+
+// Generate is called to return the results for the table at query time.
+// Constraints for generating can be retrieved from the queryContext.
+func Generate(ctx context.Context, queryContext table.QueryContext) ([]map[string]string, error) {
+	uid, gid, err := getConsoleUidGid()
+	if err != nil {
+		return nil, fmt.Errorf("failed to get console user: %w", err)
+	}
+
+	ctx, cancel := context.WithTimeout(ctx, 5*time.Second)
+	defer cancel()
+	cmd := exec.CommandContext(ctx, "defaults", "read", "/Library/Preferences/com.apple.locationmenu.plist", "ShowSystemServices")
```

Comment:
Looks like there's a setting for it.

Have we checked that it can be set via MDM? (via a `.mobileconfig` file) 

Also, if `managed_policies`+`.mobileconfig` doesn't work it seems we can check with the existing osquery `plist` table:
```sql
SELECT 1 FROM plist WHERE path='/Library/Preferences/com.apple.locationmenu.plist' AND key='ShowSystemServices' AND value=1;
```

Reply:
That's right. My bad here.
Could probably work with just this query.

Reply:
We'll probably not need this PR.